FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+MessageSend.swift

```diff
@@ -54,8 +54,14 @@ extension ComposeViewController {
     }
 
     private func sendMessage(isPlain: Bool) async throws -> MessageIdentifier {
+        // Refetch client configuration before sending password protected message
+        // https://github.com/FlowCrypt/flowcrypt-ios/issues/2658
+        let latestClientConfiguration = (!isPlain && contextToSend.hasRecipientsWithoutPubKey)
```

Comment:
let's also add check that it's enterprise build, as for now re-fetch is needed only for enterprise version

Reply:
Ohh, Yep!. Missed it.
Just added check